services/ui-src/src/containers/LocalLogin.js

```diff
@@ -14,14 +14,16 @@ export default function Login() {
       firstName: "Alice",
       lastName: "Cooper",
       lastLogin: "01/28/2021",
-      role: "admin",
+      role: "state",
+      // role:
+      //   "cn=CARTS_Group_Dev,ou=Groups,dc=cms,dc=hhs,dc=gov,cn=CARTS_Group_Dev_Admin,ou=Groups,dc=cms,dc=hhs,dc=gov,cn=CHIP_D_USER_GROUP_ADMIN,ou=Groups,dc=cms,dc=hhs,dc=gov,cn=IDM_OKTA_TEST,ou=Groups,dc=cms,dc=hhs,dc=gov",
       dateJoined: "01/22/2021",
-      isSuperUser: true,
-      isActive: true,
+      isSuperUser: "true",
+      isActive: "true",
       userId: "1",
       email: "alicecooper@collabralink.com",
       username: "AAAA",
-      states: ["AL, MD, PA"],
+      states: "AL-MD-PA",
```

Comment:
out of curiosity, why is the states in the string format, why not array like it was before, makes way more sense. and is easier to work with

Reply:
For ease of working with Dynamo, doing a string here is ok. It's easily converted to an array if needed in that format but we can parse from a string just as easily as going through an array.